### ISSUE TITLE
Bootstrap_package 12.0.1 not working with theme made using Sitepackage-Builder version10.4 

### ISSUE THREAD

**Issue by Tonyt8** [benjaminkott/bootstrap_package#1072]
Hi,

Typo3 10.4.20
My custom theme templates are made using Typo3 sitepackage-builder version 10.4. _Thanks for providing the sitepackage-builder, it's another great tool._
I use the following Bootstrap package static templates:
Full Package
Bootstrap 4.x (SCSS)
Content Elements

After updating to the bootstrap_package to 12.0.1 from 11.0.3 my custom theme templates do not work. 
Reading the error message I thought was it because the theme.scss is now in a different directory. I think I solved the theme not being found, but then I only get an error code number which do not understand. 

I have tried various things and the example from the question (Recommended way to use typoscript settings from BS4 instead of default BS5) 

In short I cannot find the solution. 

What do I need to change in my custom theme extension for it to work with bootstrap_package 12.0.1?

Thanks.

Regards,
Tony

**Comment by gilbertsoft** [comment]
Please check the recommendation in #1045

**Comment by gilbertsoft** [comment]
Try to use the core syntax for the import instead: `EXT:bootstrap_package/Resources/Public/Scss/bootstrap4/theme`

See also https://github.com/benjaminkott/bootstrap_package/blob/master/Tests/Packages/demo_package/Resources/Private/Scss/CoreSyntax/theme.scss
